src/components/card/card.tsx

```diff
@@ -1,27 +1,28 @@
-import React from 'react';
 import { ICard } from '../../interfaces/ICardInterfaces';
 import classes from './card.module.scss';
 
 interface IOneCard {
   key: number;
   card: ICard;
+  onShowModal: (id: number) => void;
 }
-class Card extends React.Component<IOneCard, object> {
-  render() {
-    return (
-      <div className={classes.cardItem}>
-        <img className={classes.cardImg} src={this.props.card.image}></img>
-        <div className={classes.descriptionCard}>
-          <p className={classes.cardTitle}>{this.props.card.name}</p>
-          <p className={classes.cardDescription}>{this.props.card.description}</p>
-          <div className={classes.cardCost}>
-            <p className={classes.cardPrise}>{this.props.card.cost + ' '}</p>
-            <span className={classes.cash}>$</span>
-          </div>
-        </div>
+const Card = (oneCard: IOneCard) => {
```

Comment:
there are several points:
1. key prop is for React, no need to add type to it
2. Destructure your props instead of using like object
3. Make more simple event handlers)
```
interface IOneCard {
  card: ICard;
  onShowModal: (id: number) => void;
}
const Card = ({ card, onShowModal }: IOneCard) => {
  const handleClick = () => {
    onShowModal(card.id);
  };

  return (
    <div className={classes.cardItem} onClick={handleClick}>
      <img className={classes.cardImg} src={card.image}></img>
      <div className={classes.descriptionCard}>
        <p className={classes.cardTitle}>{card.name}</p>
      </div>
    </div>
  );
};
```